README.md

```diff
@@ -34,7 +34,7 @@ Take ownership of your Twitter data. First talked about at [Jamstack Conf 2019](
 ### Populate the database from your Twitter Archive zip
 
 1. Copy `./data/tweets.js` from your [Twitter Archive](https://help.twitter.com/en/managing-your-account/how-to-download-your-twitter-archive) `zip` file into the `./database` directory of this project.
-   * Rename `window.YTD.tweet.part0` in `tweets.js` to `module.exports`
+   * Rename `window.YTD.tweets.part0` in `tweets.js` to `module.exports`
```

Comment:
I think it would be better writing both versions, if people copy/paste it to find it in their file, as some might have an old archive.